config/management/genesis/src/genesis.rs

```diff
@@ -41,15 +41,16 @@ impl Genesis {
     pub fn execute(self) -> Result<Transaction, Error> {
         let layout = self.layout()?;
         let libra_root_key = self.libra_root_key(&layout)?;
+        let treasury_root_key = self.treasury_root_key(&layout)?;
         let operator_assignments = self.operator_assignments(&layout)?;
         let operator_registrations = self.operator_registrations(&layout)?;
 
         let genesis = vm_genesis::encode_genesis_transaction(
             libra_root_key,
+            treasury_root_key,
             &operator_assignments,
             &operator_registrations,
-            // TODO: swap back by 8/15
```

Comment:
Hehe, that has come and passed.

Reply:
of course, all todos with a date always get cleaned up lifetimes later... and of course @zekun000 is going to force me to make this into another PR :P